.storybook/preview.js

```diff
@@ -23,34 +16,33 @@ import builtTokens from '../lib/styles/build/css/design_tokens.css';
 const cssReq = require.context('!!raw-loader!../lib/components', true, /.\.css$/);
 const cssTokenFiles = cssReq
   .keys()
-  .map(filename => ({ filename, content: cssReq(filename).default }));
+  .map((filename) => ({ filename, content: cssReq(filename).default }));
 
-addParameters({
+export const decorators = [
+  (Story) => (
+    <div className={styles.container}>
+      <Story />
+    </div>
+  ),
+];
+
+export const parameters = {
   // configure storybook-design-token
   designToken: {
     files: {
       css: cssTokenFiles,
-    }
+    },
   },
-  
+
   options: {
-    // configure @storybook/theming
-    theme: virtuosoTheme,
-
-    /**
-     * id to select an addon panel
-     * @type {String}
-     */
-    selectedPanel: 'storybook/a11y/panel',
-  }
-});
-
-addDecorator(withA11y);
-
-addDecorator(Story => (
-  <div className={styles.container}>
-    <Story />
-  </div>
-));
-
-addDecorator(withKnobs);
\ No newline at end of file
+    // why? sort the stories https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories
+    storySort: {
+      order: [
+        'Intro',
+        ['Introduction', 'Design Tokens'],
+        'Library',
+        ['Layout', 'Atoms', 'Molecules', 'Organisms'],
+      ],
+    },
```

Comment:
nice, I prefer this syntax

Reply:
Yea this is so much cleaner. It's strange at first, but the docs explain is fine enough.